app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,130 @@
+class OrdersController < ApplicationController
```

Comment:
From @lacuchilla:
The code in the index method is a bit difficult to read. I'm not sure what is happening there.
There's some commented out code in the orders controller that could be deleted to make the methods a bit cleaner.
There are a several large methods that could be broken up